test/fixtures/type-generator/query-with-defer.graphql

```diff
@@ -2,16 +2,16 @@ query TestDefer @raw_response_type {
   node(id: "1") {
     ... on User {
       name
-      friends(first: 10) {
+      friends(first: 10)
+        @stream_connection(key: "TestDefer_friends", initial_count: 0) {
         edges {
-          ... on FriendsConnection @defer {
-            pageInfo {
-              endCursor
-              hasNextPage
+          node {
+            actor {
+              name
             }
           }
         }
       }
     }
   }
-}
+}
```

Comment:
This fixture currently doesn’t look like the upstream one, what gives? https://github.com/facebook/relay/blob/v7.0.0/packages/relay-compiler/transforms/__tests__/fixtures/relay-defer-stream-transform/query-with-defer.graphql